src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -259,6 +259,16 @@
                 </value>
             </valueSetDefinition>
         </valueSet>
+    </fields>
+        <fields>
+        <fullName>EndDate__c</fullName>
```

Comment:
@andrewyu-salesforce Thanks for moving the field to this file. I have one more little request (and one that probably should have ended up in the AC's for post-merge testing). Since this is a new field, when deployed into our existing customer base as part of the Managed Package, the default will be that no Users at all have access to the new field. The partial work-around we've been using with RD2 specific fields is to deploy permissions to new fields as part of the unpackaged metadata for the System Administrator and Standard User profiles. 

With that in mind, can you update the two profile files in this [folder](https://github.com/SalesforceFoundation/NPSP/tree/master/unpackaged/config/rd2_post_config/profiles) to add Read and Edit permissions for this new field and include those changes in this PR?